gammapy/background/__init__.py

```diff
@@ -10,3 +10,4 @@
 from .template import *
 from .kernel import *
 from .models import *
+from .make import *
```

Comment:
I especially don't like `make.py`, because a year from now there will be lots of other code that makes things in `gammpay.background`, e.g. reflected regions or exclusion regions or ... and that code shouldn't be in the same file as what's in `make.py` now.

So either `cube_model.py` like I suggested, or `cube.py` like you did seems better to me.

But in the end ... we will accumulate functionality for a while and then try to do a consistent naming across Gammapy before the 1.0 release. So it's not terribly important what we choose now ... you decide!